README.md

```diff
@@ -114,22 +128,42 @@ func main() {
 }
 ```
 
-### Run FileUploader
+**1. Create a test file containing data:**
+
+On Linux systems, you may do this with `dd`:
```

Comment:
Can anybody recommend an equivalent command for Windows? 

`/dev/urandom` exists on Mac although specific OS version and user permissions might cause trouble?

Reply:
```
fsutil file createnew <file name/path/extension> <size of file in bytes>
```

```
fsutil file createnew "C:\Users\<username>\Desktop\sample.txt" 104857600
```

Reply:
> fsutil file createnew ...

Thank you @djwfyi !!